docker-rancher/server.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
```

Comment:
This parameters file is not needed. Only the main azuredeploy and azuredeploy.parameters should be able to run the entire template